CHANGELOG.md

```diff
@@ -105,6 +105,7 @@
 * [ENHANCEMENT] Add support for autoscaling distributors. #3378
 * [ENHANCEMENT] Make auto-scaling logic ensure integer KEDA thresholds. #3512
 * [BUGFIX] Fixed query-scheduler ring configuration for dedicated ruler's queries and query-frontends. #3237 #3239
+* [BUGFIX] Jsonnet: Fix auto-scaling so that ruler-querier CPU threshold is a string-encoded integer millicores value.
```

Comment:
```suggestion
* [BUGFIX] Jsonnet: Fix queriers auto-scaling when requested CPU is a fraction of a core. #3520
```

Reply:
@pracucci I think the above suggestion should be "ruler-queriers".